codice/ddf
# DDF-3729 Added documentation for Query Feedback

#### What does this PR do?
This PR adds documentation for Query Feedback and also updates the documentation for `Catalog UI Search` and `Standard Search UI` to match the `Admin Console`. The last commit also removes the `Configuring the Standard Search UI` section because the only subsection included incorrect content.
#### Who is reviewing it? 
@kcover @troymohl 
#### Select relevant component teams: 
@codice/docs 
#### Choose 2 committers to review/merge the PR. 
@clockard
@ricklarsen - Documentation
#### How should this be tested?

- Review the updated documentation, HTML and PDF ([attached](https://github.com/codice/ddf/files/1956106/documentation.zip)).
- Test the updated `Catalog UI Search` and `Standard Search UI` configuration modals in the `Admin Console`.
  - Check that the titles and tooltips are formatted correctly.
  - Confirm that saving the `Standard Search UI` without specifying the `Imagery Providers` does not log a warning.
  - Confirm that configuring the `Standard Search UI` with a valid imagery provider works.

#### Any background context you want to provide?
See https://github.com/codice/ddf/pull/1858 for more details about the Query Feedback Feature.
#### What are the relevant tickets?
[DDF-3729](https://codice.atlassian.net/browse/DDF-3729)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/8041246/39154357-c2ee3982-4702-11e8-96ad-fd26cc073b7d.png)
![image](https://user-images.githubusercontent.com/8041246/39266743-bdf6d29e-487f-11e8-9705-646b0ac72b1c.png)
#### Checklist:

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
